packages/zoe/src/zoeService/zoeStorageManager.js

```diff
@@ -212,7 +212,7 @@ export const makeZoeStorageManager = (
             keyword,
             assetKind,
             displayInfo,
-            state.adminNode.terminateWithFailure,
+            E(state.adminNode).terminateWithFailure,
```

Comment:
Can we pass a method wrapped with E?  I  suspect this needs to pass a thunk instead.

Reply:
You're right that this won't work. 

```suggestion
            reason => E(state.adminNode).terminateWithFailure(reason),
```
The old code should not have worked either. Did it?

Reply:
I don't think so. I ran into one case and tested the fix for it. The others were find-replace. I'll make this correction and rebase.